channels/smartcard/client/smartcard_pack.c

```diff
@@ -3868,7 +3868,7 @@ LONG smartcard_pack_read_cache_return(SMARTCARD_DEVICE* smartcard, wStream* s,
                                       const ReadCache_Return* ret)
 {
 	LONG status;
-	DWORD index = 0;
+	UINT32 index = 0;
```

Comment:
This should not be necessary, `UINT32` and `DWORD` are both defined to the same 32bit unsigned integer type, if not then something in winpr is missing for MinGW